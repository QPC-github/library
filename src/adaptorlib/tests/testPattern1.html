<HTML>
<HEAD></head>
<BODY>
Today, John Paul gave a speach at local animal shelter. Animal lovers rejoice.
</BODY>
</HTML>
