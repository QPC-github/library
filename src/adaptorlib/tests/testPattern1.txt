John\sPaul <meta name="pope" content="John Paul 2nd" />
local <meta name="city" content="Mountain View" />
