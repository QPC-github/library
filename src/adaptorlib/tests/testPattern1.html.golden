<HTML>
<HEAD>
<meta name="pope" content="John Paul 2nd" />
<meta name="city" content="Mountain View" />
</HEAD>
<BODY>
Today, John Paul gave a speach at local animal shelter. Animal lovers rejoice.
</BODY>
</HTML>
